<launch>
    <arg name="folder_name" default="$(find oct_levitation)/data/experiment_data"/>
    <arg name="bagfile_name" default="experiment_data.bag"/>
    <rosparam command="load" file="$(find oct_levitation)/config/experiment_analysis_params.yaml"/>
    <node pkg="rosbag" type="record" name="rosbag_record" output="screen" args="-a -O $(arg folder_name)/$(arg bagfile_name)"/>
</launch>